src/cron/hourly-mail.ts

```diff
@@ -0,0 +1,7 @@
+import * as cron from 'node-cron';
+import {sendEmailToUser} from '../services/mail-service'
+
+cron.schedule('0 * * * *', () => {
```

Comment:
Actually, the cronjob will be handled in Kubernetes for now. 
So it is unnecessary and we can delete this file (maybe in the future, we will restore it).